### PR TITLE
Read bazel lib version using bazel CLI instead of hardcoding it.

### LINKED ISSUE #74: Use the MODULE.bazel version string and avoid hardcoding it into build.rs files.
## Desired behavior

Avoid this:

https://github.com/maliput/maliput-rs/blob/76ad45a1da4aba55c2d028ab542c3530657df75f/maliput-sdk/build.rs#L60-L61

by reading the MODULE.bazel file and scrapping the version.

## Alternatives considered
- Consider looking for a variable in the file.
- Make query to bazel (mod) and get the package and version. 

## Implementation suggestion

N/A

## Additional context
N/A

### PR DESCRIPTION
**Pull request by agalbachicar**
# 🎉 New feature

Closes #74

## Summary
Executes `bazel mod explain` to obtain `maliput` and `maliput_malidrive` library versions in `maliput-sdk/build.rs`.

## Test it
By executing cargo check / build.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if it affects the public API)

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Review by francocipollone — APPROVED** [review]
LGTM. Check clippy. CI is also failing weirdly.

**Comment by francocipollone** [comment]
One interesting thing happened:  CC @agalbachicar 
Remember that the `print` is mainly used at build.rs for indicating to `cargo` some stuff (links,rustc envs, etc)

You were using `print` macro for printing information (with no new_line)  so if we check the output of the maliput-sdk build:

```
cargo:rerun-if-changed=src/lib.rs
cargo:rerun-if-changed=BUILD.bazel
cargo:rerun-if-changed=MODULE.bazel
Detected maliput version: <1.2.0>Detected maliput_malidrive version: <0.2.1>cargo:CXXBRIDGE_DIR0=/workspaces/maliput-rs/target/debug/build/maliput-sdk-bdf1a05400489123/out/bazel_output_base/bazel-bin/external/maliput~1.2.0/_virtual_includes/geometry_base
cargo:CXXBRIDGE_DIR1=/workspaces/maliput-rs/target/debug/build/maliput-sdk-bdf1a05400489123/out/bazel_output_base/bazel-bin/external/maliput~1.2.0/_virtual_includes/api
cargo:CXXBRIDGE_DIR2=/workspaces/maliput-rs/target/debug/build/maliput-sdk-bdf1a05400489123/out/bazel_output_base/bazel-bin/external/maliput~1.2.0/_virtual_includes/math
cargo:CXXBRIDGE_DIR3=/workspaces/maliput-rs/target/debug/build/maliput-sdk-bdf1a05400489123/out/bazel_output_base/bazel-bin/external/maliput~1.2.0/_virtual_includes/utility
cargo:CXXBRIDGE_DIR4=/workspaces/maliput-rs/target/debug/build/maliput-sdk-bdf1a05400489123/out/bazel_output_base/bazel-bin/external/maliput~1.2.0/_virtual_includes/drake
cargo:CXXBRIDGE_DIR5=/workspaces/maliput-rs/target/debug/build/maliput-sdk-bdf1a05400489123/out/bazel_output_base/bazel-bin/external/maliput~1.2.0/_virtual_includes/common
cargo:CXXBRIDGE_DIR6=/workspaces/maliput-rs/target/debug/build/maliput-sdk-bdf1a05400489123/out/bazel_output_base/bazel-bin/external/maliput~1.2.0/_virtual_includes/base
cargo:CXXBRIDGE_DIR7=/workspaces/maliput-rs/target/debug/build/maliput-sdk-bdf1a05400489123/out/bazel_output_base/bazel-bin/external/maliput~1.2.0/_virtual_includes/drake_private_systems
cargo:CXXBRIDGE_DIR8=/workspaces/maliput-rs/target/debug/build/maliput-sdk-bdf1a05400489123/out/bazel_output_base/bazel-bin/external/maliput~1.2.0/_virtual_includes/plugin
cargo:CXXBRIDGE_DIR9=/workspaces/maliput-rs/target/debug/build/maliput-sdk-bdf1a05400489123/out/bazel_output_base/bazel-bin/external/maliput~1.2.0/_virtual_includes/drake_private_common
cargo:rustc-env=MALIPUT_BIN_PATH=/workspaces/maliput-rs/target/debug/build/maliput-sdk-bdf1a05400489123/out/bazel_output_base/bazel-bin/external/maliput~1.2.0
cargo:rustc-env=MALIPUT_MALIDRIVE_BIN_PATH=/workspaces/maliput-rs/target/debug/build/maliput-sdk-bdf1a05400489123/out/bazel_output_base/bazel-bin/external/maliput_malidrive~0.2.1
cargo:rustc-env=MALIPUT_MALIDRIVE_PLUGIN_PATH=/workspaces/maliput-rs/target/debug/build/maliput-sdk-bdf1a05400489123/out/bazel_output_base/bazel-bin/external/maliput_malidrive~0.2.1/maliput_plugins
cargo:rustc-env=MALIPUT_MALIDRIVE_RESOURCE_PATH=/workspaces/maliput-rs/target/debug/build/maliput-sdk-bdf1a05400489123/out/bazel_output_base/bazel-bin/libmaliput_sdk.so.runfiles/maliput_malidrive~0.2.1/resources
cargo:root=/workspaces/maliput-rs/target/debug/build/maliput-sdk-bdf1a05400489123/out
cargo:maliput_bin_path=/workspaces/maliput-rs/target/debug/build/maliput-sdk-bdf1a05400489123/out/bazel_output_base/bazel-bin/external/maliput~1.2.0
cargo:maliput_malidrive_bin_path=/workspaces/maliput-rs/target/debug/build/maliput-sdk-bdf1a05400489123/out/bazel_output_base/bazel-bin/external/maliput_malidrive~0.2.1
cargo:maliput_malidrive_plugin_path=/workspaces/maliput-rs/target/debug/build/maliput-sdk-bdf1a05400489123/out/bazel_output_base/bazel-bin/external/maliput_malidrive~0.2.1/maliput_plugins

```

You see in the 4th line that the cargo:CXXBRIDGE_DIR0 command isn't parsed correctly and then the cxx configuration go bananas.

The bottom line is: Avodining regular prints in build.rs files might be the best.

**Review by francocipollone — APPROVED** [review]
LGTM!

**Comment by francocipollone** [comment]
Extra comment: @agalbachicar 
You can execute ./tools/reformat_code.sh script for making clippy and fmt happy before git commit (+presubmit)

**Comment by agalbachicar** [comment]
Thanks @francocipollone !
